cypress/page/payment-step.page.ts

```diff
@@ -6,8 +6,8 @@ class PaymentStepPage{
 
     constructor(){
         this.bankWireAHyperlink = "[class=bankwire]";
-        this.confirmPayButton = "#cart_navigation > button";
-        this.finishShoppingPTitle = "#center_column > div > p > strong";
+        this.confirmPayButton = "#cart_navigation > button.button.btn.btn-default.button-medium";
```

Comment:
if you look to the source code, the button has inside a span, you can use the `span` instead of `button.button.btn.btn-default.button-medium`.

However, take a look to: https://docs.cypress.io/api/commands/contains#Content and see if you can use the name of the button with the contains function ;)